### PR TITLE
revert changes to populate method, perhaps we need nother version of …

### PR DESCRIPTION
**Pull request by pruan-rht**
…the populate for 'apis/k8s.xxx' calls.  For reference https://github.com/openshift/verification-tests/issues/118

### CODE REVIEW

File: lib/rest_kubernetes.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@ module Rest
     module Kubernetes
       extend Helper
 
-      def self.populate(path, base_opts, opts, type)
-        populate_common("/apis/#{type}.k8s.io/<api_version>", path, base_opts, opts)
+      def self.populate(path, base_opts, opts)
+        populate_common("/api/<api_version>", path, base_opts, opts)
```
Comment (akostadinov) [inline lib/rest_kubernetes.rb]: How about
```
def self.populate(path, base_opts, opts, type: nil)
  type_str = type ? "/#{type}.k8s.io" : ""
  populate_common("/apis#{type_str}/<api_version>", path, base_opts, opts)
```

I will apply this patch now to get things fixed. But for future resolution.

Comment (pruan-rht) [reply lib/rest_kubernetes.rb]: sounds good.  It should be `type_str = type ? "s/#{type}.k8s.io" : ""` though because the other format is `apis`

**Comment by xingxingxia** [comment]
Thanks for the PR and above new solution.
Will test the case mentioned with failure log in https://github.com/openshift/verification-tests/issues/118 top comment. @zhouying7780 could you please help the test when available?

**Pull request merged**